src/HotChocolate/Core/src/Execution/Processing/SubscriptionExecutor.Subscription.cs

```diff
@@ -127,6 +127,9 @@ private async Task<IQueryResult> OnEvent(object payload)
                     return await _queryExecutor
                         .ExecuteAsync(operationContext, scopedContext)
                         .ConfigureAwait(false);
+
+                    // I believe operation context should be marked as completed here?
```

Comment:
@michaelstaib  Could you please look at this comment? I left it, because I believe that operation context should be completed here, but of course I dont understand all code mechanics. Please remove it if it is wrong.

Reply:
This is now solved. I reworked the policy to better reuse contexts and always clean them.